test/Microsoft.AspNetCore.Server.KestrelTests/ChunkedRequestTests.cs

```diff
@@ -78,6 +78,8 @@ public static TheoryData<ServiceContext> ConnectionFilterData
                          "");
                     await connection.ReceiveEnd(
                         "HTTP/1.1 200 OK",
+                        "Date: " + testContext.DateHeaderValue,
+                        "Server: Kestrel",
```

Comment:
Sounds good to me, will make the tests more concise. I don't think there are any tests where it actually matters that the `Server` header is present or that `response.Headers.Clear()` is called, but please be mindful of that.